src/reynir_correct/annotation.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         code: str,
         text: str,
         detail: Optional[str] = None,
+        references: Optional[List[str]] = [],
```

Comment:
It looks like List[str] is enough, the Optional[] is superfluous. An empty list of references is just an empty list, not None.

Reply:
Done